Gifski/Tooltip.swift

```diff
@@ -40,6 +40,11 @@ final class Tooltip: NSPopover {
 		of positioningView: NSView,
 		preferredEdge: NSRectEdge
 	) {
+		guard positioningView.window != nil else {
+			assertionFailure("Tooltip must be shown from a view with a window")
+			return
+		}
+
 		if !showOnlyOnce || (showOnlyOnce && defaults[showKey] < 1) {
 			defaults[showKey] += 1
 			super.show(relativeTo: positioningRect, of: positioningView, preferredEdge: preferredEdge)
```

Comment:
@sunshinejr Just an idea, should be wrap this in `DispatchQueue.main.async {}` to ensure it's run in the next run loop, just as a preventive measure against race issues with other animations.

Reply:
hmm, good idea, we could try!

Reply:
Thinking about this more, doing this might causes other problems, like the tooltip not being shown if a modal dialog is presented until after it's dismissed. It might be better to wrap it yourself when needed.